src/EventQuery.php

```diff
@@ -137,9 +137,11 @@ protected function init_end_date(array $params, DateTimeImmutable $start) : Date
    * Get the results for this EventQuery as a collection of zero or more Events
    *
    * @internal
+   * @return array<\Timber\Post>
    */
-  public function get_results() {
-    return Timber::get_posts($this->params());
+  public function get_results() : array {
+    $posts = Timber::get_posts($this->params());
+    return $posts ? $posts->to_array() : [];
```

Comment:
here ☝️

Reply:
call sites were updated to assume arrays.